xskillscore/core/np_deterministic.py

```diff
@@ -188,6 +189,55 @@ def _pearson_r(a, b, weights, axis, skipna):
     return res
 
 
+def _r2(a, b, weights, axis, skipna):
+    """
+    ndarray implementation of sklearn.metrics.r2_score.
+
+    Parameters
+    ----------
+    a : ndarray
+        Input array.
+    b : ndarray
+        Input array.
+    axis : int
+        The axis to apply the r2_score along.
+    weights : ndarray
+        Input array of weights for a and b.
+    skipna : bool
+        If True, skip NaNs when computing function.
+
+    Returns
+    -------
+    res : ndarray
+        R^2 (coefficient of determination) score.
+
+    See Also
+    --------
+    sklearn.metrics.r2_score
```

Comment:
mind adding: Wikipedia entry on the Coefficient of determination
            <https://en.wikipedia.org/wiki/Coefficient_of_determination>`_

Reply:
Done